test/EVMHost.cpp

```diff
@@ -249,13 +249,33 @@ evmc::result EVMHost::call(evmc_message const& _message) noexcept
 
 	if (message.kind == EVMC_CREATE)
 	{
-		// TODO this is not the right formula
 		// TODO is the nonce incremented on failure, too?
+		sender.nonce++;
```

Comment:
Shouldn't this start at 0? So the increment should be after, right?

Reply:
Yes. The address should be computed before the sender's nonce is bumped. E.g. https://github.com/ethereum/evmone/pull/484/files#diff-a6fe16f4d5493d757459aa2ee53b83d1e7761259fe171a0aed72ea950d2393d5R155-R159

Reply:
This is a bit different matter. When contract is created it has nonce 1.